### PR TITLE
errors in formulas

### PR DESCRIPTION
**Pull request by mfournial**
You have to give me 10 points for that

### CODE REVIEW

**Comment by PK398** [comment]
Wow, very impressive that you spotted and removed that comma. It brings new meaning to life. Such stupidity, Much wow.

**Comment by mfournial** [comment]
There is no small mistakes, as someone taking M1M1 I thought you'd knew it... Disappointing as usual

File: 141.tex [hunk 1]
```diff
@@ -157,7 +157,7 @@ \subsection{Induction over Recursively Defined Structures}
 For a property $Q\subseteq\mathbb{N}\times\mathbb{N}$, we get the
 inductive principle
 \begin{multline*}
-\forall k\in\mathbb{N}.Q\left(0,k+1\right)\land\forall m,n\in\mathbb{N}.,\left[SL\left(m,n\right)\land Q\left(m,n\right)\rightarrow Q\left(m+1,n+1\right)\right]\\
+\forall k\in\mathbb{N}.Q\left(0,k+1\right)\land\forall m,n\in\mathbb{N}.\left[SL\left(m,n\right)\land Q\left(m,n\right)\rightarrow Q\left(m+1,n+1\right)\right]\\
```
Comment (lsquires) [inline 141.tex:160]: Nice

Comment (PK398) [reply 141.tex:160]: not nice

Comment (mfournial) [reply 141.tex:160]: *essential and vital

**Pull request merged**